tico/passes/remove_redundant_permute.py

```diff
@@ -45,9 +61,13 @@ def __init__(self):
     def call(self, exported_program: ExportedProgram) -> PassResult:
         """
         [BEFORE]
-            (AxBxC) - aten.permute - aten.permute - (AxBxC)
+            (AxBxC) - aten.permute_1 - aten.permute_2 - (OUT_SHAPE)
         [AFTER]
-            (AxBxC)
+            if OUT_SHAPE == (AxBxC):
+                (AxBxC)
+            else:
+                (AxBxC) - aten.permute (fused dims) - (OUT_SHAPE)
```

Comment:
Why don't we add the case where `permute_1` and `permute_2` are removed to the explanation? 
I think it can be misunderstood as if it exists only in the case of fused.